MAUI/Rotator/Dots-Customization.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: Dots Customization in .NET MAUI Rotator control | Syncfusion
+description: Learn about Dots Customization support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
+platform: maui 
+control: Rotator 
+documentation: ug
+---
+
+# Dots Customization in .NET MAUI Rotator (SfRotator)
+
+The Rotator control supports customizing dots' border color, selected dots' color, and unselected dots' color.
+
+## DotsBorder Color
+
+The `DotsBorderColor` property is used to customize the color of the dots border in the `SfRotator`.
```

Comment:
Use DotsStroke

Reply:
changed to DotsBorderColor  to  DotsStroke